easy_infra.yml

```diff
@@ -31,7 +31,7 @@ packages:
           KICS_EXCLUDE_SEVERITIES: --exclude-severities
           KICS_INCLUDE_QUERIES: --include-queries
         description: directory scan
-    version: 2.9.6+dfsg-1
+    version: 2.10.7+merged+base+2.10.8+dfsg-1
```

Comment:
Make sure to test `task update`

Reply:
task update functioned after my changes above. Although it reset the two version changes here back to their original values.